From 5030b789a9485e6f8ed77a61a6e1f8820b6c3456 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 8 Nov 2023 11:23:47 +0100
Subject: [PATCH 0599/1141] ARM: dts: stm32: avoid scmi regulator yaml warning
 for stm32mp131

Avoid an SCMI YAML code checking error.
The SCMI YAML oblige the node name to be regulator@xx.

Change-Id: Ib994489920c94d41b8c14ae41633012b872af420
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/338454
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp131.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/stm32mp131.dtsi b/arch/arm/boot/dts/stm32mp131.dtsi
index f86c5d5a47f8..e830d068d5f1 100644
--- a/arch/arm/boot/dts/stm32mp131.dtsi
+++ b/arch/arm/boot/dts/stm32mp131.dtsi
@@ -104,15 +104,15 @@ scmi_regu: regulators {
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					scmi_reg11: voltd-reg11 {
+					scmi_reg11: regulator@0 {
 						reg = <VOLTD_SCMI_REG11>;
 						regulator-name = "reg11";
 					};
-					scmi_reg18: voltd-reg18 {
+					scmi_reg18: regulator@1 {
 						reg = <VOLTD_SCMI_REG18>;
 						regulator-name = "reg18";
 					};
-					scmi_usb33: voltd-usb33 {
+					scmi_usb33: regulator@2 {
 						reg = <VOLTD_SCMI_USB33>;
 						regulator-name = "usb33";
 					};
-- 
2.39.2

